RecoHI/HiEvtPlaneAlgos/python/HiEvtPlane_cfi.py

```diff
@@ -1,18 +1,27 @@
 import FWCore.ParameterSet.Config as cms
 
 hiEvtPlane = cms.EDProducer("EvtPlaneProducer",
-                            vtxCollection_=cms.InputTag("hiSelectedVertex"),
-                            caloCollection_=cms.InputTag("towerMaker"),
-                            trackCollection_=cms.InputTag("hiGeneralTracks"),                           
-                            useECAL_ = cms.untracked.bool(True),
-                            useHCAL_ = cms.untracked.bool(True),
-                            useTrackPtWeight_ = cms.untracked.bool(True),
-                            minet_ = cms.untracked.double(0.2),
-                            maxet_ = cms.untracked.double(500.0),
+                            vertexTag_=cms.InputTag("hiSelectedVertex"),
+                            centralityTag_=cms.InputTag("hiCentrality"),
+                            caloTag_=cms.InputTag("towerMaker"),
+                            castorTag_=cms.InputTag("CastorTowerReco"),
+                            trackTag_=cms.InputTag("hiGeneralTracks"),         
+                            centralityBinTag_ = cms.InputTag("centralityBin","HFtowers"),
+                            centralityVariable = cms.string("HFtowers"),
+                            nonDefaultGlauberModel = cms.string(""),
+                            FlatOrder_ = cms.untracked.int32(9),
```

Comment:
all parameters that can change physics results (values of stored parameters) have to be tracked.
At a quick glance, essentially all parameters in this config have to be tracked